javascript/ql/src/Security/CWE-116/IncompleteHtmlAttributeSanitization.ql

```diff
@@ -0,0 +1,27 @@
+/**
+ * @name Incomplete HTML attribute sanitization
+ * @description Writing incompletely sanitized values to HTML
+ *              attribute strings allows for a cross-site scripting
+ *              vulnerability.
+ * @kind path-problem
+ * @problem.severity warning
+ * @precision high
+ * @id js/incomplete-html-attribute-sanitization
+ * @tags security
+ *       external/cwe/cwe-079
+ *       external/cwe/cwe-116
+ *       external/cwe/cwe-20
+ */
+
+import javascript
+import DataFlow::PathGraph
+import semmle.javascript.security.dataflow.IncompleteHtmlAttributeSanitization::IncompleteHtmlAttributeSanitization
+import Security.IncompleteBlacklistSanitizer::Util
```

Comment:
That's an unusual place for a shared library. What's wrong with `semmle.javascript.security`? I'm not a huge fan of the `semmle.javascript` prefix but let's at least be consistent.